Makefile

```diff
@@ -66,6 +66,14 @@ kubevirt-sync:
 kubevirt-functest:
 	./scripts/kubevirt.sh functest
 
+.PHONY: functest
+functest: kubevirt-sync
```

Comment:
Can you remove `kubevirt-sync` please? I'd keep this a manual step. (can be `cluster-sync` or `kubevirt-sync`)

Reply:
And can you please re-use the existing `kubevirt-functest` and `cluster-functest` targets? I would also move setting `KUBECONFIG` into these files.